src/main/webapp/createYourOwn.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```

Comment:
Your file naming is not consistent. For example `create-your-own-style.css` vs `createYourOwn.html`. Maybe there should be one convention: say all lower case letters, words separated with `_` characters?

Reply:
ok, I will put all of the file names with the uppercase letters convention like: createYourOwn